snippets/rust.json

```diff
@@ -416,6 +416,13 @@
         ],
         "description": "impl … { … }"
     },
+    "inline": {
+        "prefix": "inline",
+        "body": [
+            "#[inline]"
```

Comment:
Sounds good, but I think `inlinefn` is better than.
What do you think about it?